app/controllers/overrides/registrations_controller.rb

```diff
@@ -36,6 +37,14 @@ def error_checks_for_create
       # success redirect url is required
     end
 
+    def name_fields_have_invalid_characters?
+      includes_url_characters(params[:contact][:firstName]) || includes_url_characters(params[:contact][:lastName]) || includes_url_characters(params[:contact][:middleName])
```

Comment:
suggestion: Concatenate values, e.g.
```ruby
includes_url_characters(params[:contact].values.join)
```

Reply:
I decided we don't need to include `middleName` because it is not included in the email template, and since it is an optional field.

Reply:
A string like `bitcoin.ru/get-rich` would pass the checks, but might still be something that would be auto-linked in an email client.

Reply:
We can add the `.` check back if we think that is needed.